pkg/gui/presentation/icons/file_icons.go

```diff
@@ -5,327 +5,327 @@ import (
+	".jsx":            {"\ue7ba", 45},      // 
+	".jxl":            {"\uf1c5", 241},     // 
+	".ksh":            {"\ue795", 241},     // 
+	".kt":             {"\ue634", 99},      // 
+	".kts":            {"\ue634", 99},      // 
+	".latex":          {"\uf034", 241},     // 
+	".less":           {"\ue758", 54},      // 
+	".lhs":            {"\ue777", 140},     // 
+	".license":        {"\U000f0219", 185}, // 󰈙
+	".localized":      {"\uf179", 15},      // 
+	".lock":           {"\uf023", 241},     // 
+	".log":            {"\uf18d", 188},     // 
+	".lua":            {"\ue620", 74},      // 
+	".lz":             {"\uf410", 241},     // 
+	".lz4":            {"\uf410", 241},     // 
+	".lzh":            {"\uf410", 241},     // 
+	".lzma":           {"\uf410", 241},     // 
+	".lzo":            {"\uf410", 241},     // 
+	".m":              {"\ue61e", 111},     // 
+	".mm":             {"\ue61d", 111},     // 
+	".m4a":            {"\uf001", 239},     // 
+	".markdown":       {"\uf48a", 74},      // 
+	".md":             {"\uf48a", 74},      // 
+	".mdx":            {"\uf48a", 74},      // 
+	".mjs":            {"\ue74e", 185},     // 
+	".mk":             {"\ue795", 241},     // 
+	".mkd":            {"\uf48a", 74},      // 
+	".mkv":            {"\uf03d", 241},     // 
+	".mobi":           {"\ue28b", 241},     // 
+	".mov":            {"\uf03d", 241},     // 
+	".mp3":            {"\uf001", 241},     // 
+	".mp4":            {"\uf03d", 241},     // 
+	".msi":            {"\ue70f", 241},     // 
+	".mustache":       {"\ue60f", 241},     // 
+	".nix":            {"\uf313", 241},     // 
+	".node":           {"\U000f0399", 197}, // 󰎙
+	".npmignore":      {"\ue71e", 197},     // 
+	".odp":            {"\uf1c4", 241},     // 
+	".ods":            {"\uf1c3", 241},     // 
+	".odt":            {"\uf1c2", 241},     // 
+	".ogg":            {"\uf001", 241},     // 
+	".ogv":            {"\uf03d", 241},     // 
+	".otf":            {"\uf031", 241},     // 
+	".part":           {"\uf43a", 241},     // 
+	".patch":          {"\uf440", 241},     // 
+	".pdf":            {"\uf1c1", 241},     // 
+	".php":            {"\ue73d", 61},      // 
+	".pl":             {"\ue769", 241},     // 
+	".png":            {"\uf1c5", 241},     // 
+	".ppt":            {"\uf1c4", 241},     // 
+	".pptx":           {"\uf1c4", 241},     // 
+	".procfile":       {"\ue21e", 241},     // 
+	".properties":     {"\ue60b", 185},     // 
+	".ps1":            {"\ue795", 241},     // 
+	".psd":            {"\ue7b8", 241},     // 
+	".pxm":            {"\uf1c5", 241},     // 
+	".py":             {"\ue606", 214},     // 
+	".pyc":            {"\ue606", 214},     // 
+	".r":              {"\uf25d", 241},     // 
+	".rakefile":       {"\ue21e", 241},     // 
+	".rar":            {"\uf410", 241},     // 
+	".razor":          {"\uf1fa", 241},     // 
+	".rb":             {"\ue21e", 241},     // 
```

Comment:
There's quite a few greys here. Is there anywhere online with a more complete list of colours?

Reply:
I  changed some more of them, the list I was working from didn't have many of the icons lazygit supports, I went through trying to match the colors to there relevant logos for the rest of them, some will have to be grey. I also fixed the structs being converted to string. @jesseduffield